articles/aks/node-updates-kured.md

```diff
@@ -69,7 +69,7 @@ kubectl create namespace kured
 helm install my-release kubereboot/kured --namespace kured --set controller.nodeSelector."kubernetes\.io/os"=linux
 ```
 
-You can also configure extra parameters for `kured`, such as integration with Prometheus or Slack. For more information about configuration parameters, see the [kured Helm chart][kured-install].
+You can also configure extra parameters for `kured`, such as integration with Prometheus or Slack. For more information about configuration parameters, see the [kured Helm chart](https://github.com/kubereboot/charts/tree/main/charts/kured).
```

Comment:
@thepaulmacca - To maintain consistency with our link reference standard/style, please update line 111 with the correct link reference and revert back to [kured-install] as it was previously. Thanks!

Reply:
I don't understand this request. I'm adding the correct link to the helm chart

- `[kured-install]` doesn't have a link attached to it, unless I'm missing something

Reply:
How are these links constructed here? Share are the actual URLs?

Reply:
@MGoedtel I've updated now if you want to check again